stl/inc/chrono

```diff
@@ -473,10 +473,11 @@ namespace chrono {
     private:
         unsigned char _Weekday;
 
-        // courtesy of Howard Hinnant
+        // courtesy of Howard Hinnant (modified to avoid overflow)
         // https://howardhinnant.github.io/date_algorithms.html#weekday_from_days
         _NODISCARD static constexpr unsigned int _Weekday_from_days(int _Tp) noexcept {
-            return static_cast<unsigned int>(_Tp >= -4 ? (_Tp + 4) % 7 : (_Tp + 5) % 7 + 6);
+            const auto _Before_modulo = static_cast<unsigned int>(_Tp) + (_Tp >= 0 ? 4u : 0u);

```

Comment:
The 3rd operand of the ternary operator is `0` provided that `int`/`unsigned int` are 32 bits long. I don't know if the STL supports any system where this is not true. If so or if you wish to make the code future proof, then the compiler can calculate the operand for us (see a simulation [here](https://godbolt.org/z/b4fz9EsKG)):
```suggestion
            constexpr auto minus_1_mod7 = static_cast<unsigned int>(-1) % 7;
            constexpr auto offset = (minus_1_mod7 >= 3 ? 3 : -4) - minus_1_mod7;
            const auto before_modulo = static_cast<unsigned int>(_Tp) + (_Tp >= 0 ? 4 : offset);
```
Otherwise, the code is good as it is.